### PR TITLE
Fix dummy loss and metrics in odps_iris_dnn_model

### PR DESCRIPTION
**Pull request by mhaoli**
Fixes dummy loss and metrics in `model_zoo/odps_iris_dnn_model.py`.

### CODE REVIEW

**Review by terrytangyuan — CHANGES_REQUESTED** [review]
Need further testing

**Comment by mhaoli** [comment]
> Need further testing

Done. Test with the following command:

```
python -m elasticdl.python.elasticdl.client train \
    --image_base=elasticdl:ci \
    --docker_image_repository=reg.docker.alibaba-inc.com/chris_lmh/ \
    --model_zoo=model_zoo \
    --model_def=odps_iris_dnn_model.odps_iris_dnn_model.custom_model \
    --cluster_spec=/Users/lmh/Workspace/EDL_workspace/sigma.py \
    --training_data=sqlflow_test_iris_train \
    --evaluation_data=sqlflow_test_iris_train \
    --data_reader_params='columns=["sepal_length", "sepal_width", "petal_length", "petal_width", "class"]' \
    --num_epochs=5 \
    --master_resource_request="cpu=1,memory=4096Mi,ephemeral-storage=10240Mi" \
    --worker_resource_request="cpu=1,memory=4096Mi,ephemeral-storage=10240Mi" \
    --minibatch_size=32 \
    --evaluation_steps=4 \
    --num_minibatches_per_task=1 \
    --num_workers=1 \
    --grads_to_wait=2 \
    --job_name=test-iris-custom-model-odps-lmh-8 \
    --log_level=DEBUG \
    --image_pull_policy=Always \
    --envs="ODPS_PROJECT_NAME=***,ODPS_ACCESS_ID=***,ODPS_ACCESS_KEY=***"\
    --namespace=kubemaker
```

**Review by terrytangyuan — APPROVED** [review]
Thanks!
